pkg/workflow/controller/handlers/workflowtrigger/cron_job.go

```diff
@@ -16,6 +18,11 @@ import (
 
 const (
 	KeyTemplate = "%s/%s"
+
+	// the time zone difference, in minutes, from current locale (host system settings) to UTC
+	// -480 for Asia/Shanghai
+	ParamTimeZoneOffset = "timezone"
```

Comment:
I'd prefer the parameter name `timezone` to be change to `timezoneOffset`, it will be set by users of Cyclone Workflow Engine, but `timezone` is not accurate.